glass/fields.py

```diff
@@ -35,36 +34,22 @@
 import numpy.typing as npt
 from gaussiancl import gaussiancl
 
-# types
-Size = typing.Optional[typing.Union[int, tuple[int, ...]]]
-Iternorm = tuple[typing.Optional[int], npt.NDArray[typing.Any], npt.NDArray[typing.Any]]
-ClTransform = typing.Union[
-    str, typing.Callable[[npt.NDArray[typing.Any]], npt.NDArray[typing.Any]]
-]
-Cls = collections.abc.Sequence[
-    typing.Union[npt.NDArray[typing.Any], collections.abc.Sequence[float]]
-]
-Alms = npt.NDArray[typing.Any]
```

Comment:
I got rid of these as I don't really like custom types. Can return if there are strong feelings.

Reply:
Although I suspect `mypy` won't pass if all have the same type, i.e. `Cls` do vary depending on context